.npm-upgrade.json

```diff
@@ -0,0 +1,108 @@
+{
+  "ignore": {
+    "@formatjs/intl-pluralrules": {
+      "versions": "",
+      "reason": "MM-34912"
```

Comment:
Maybe include a one line description of why here? That way you can look though this list and quickly check if the reason is something you should check up on.

Reply:
Good call. I went through and added those.